src/plugins/intel_npu/src/plugin/npuw/partitioning/online/group.cpp

```diff
@@ -375,10 +388,32 @@ std::unordered_set<Interconnect> Group::interconnect(const Group::GPtr& gptr_pro
     return ics;
 }
 
+std::string Group::specialTags() const {
+    std::string tags = "";
+
+    if (m_nofold) {
+        tags += "nofold";
+    }
+
+    if (!m_isol_tag.empty()) {
+        tags += m_isol_tag;
+    }
+
+    return tags;
+}
```

Comment:
Somewhat often. But these special tags might change after each fusion since a group takes other's flags. If you were thinking about making it static I don't see how it's possible

Reply:
Profiling will tell